normandy/recipes/models.py

```diff
@@ -94,7 +108,9 @@ def enable(self):
 
     def disable(self, ignore_revision_id=False, *args, **kwargs):
         self.enabled = False
-        self.approver = None
+        self.approval = None
```

Comment:
I'm not sure if disabling a recipe should automatically remove it's approval. For example, if we want to update the action code, we have to disable all recipes using an action, update it, and then re-enable them. In other cases we might want to temporarily disable a recipe and then re-deploy it.

I guess the real question here is: What is approval approving of? Is it simply reviewing a given configuration for a recipe and verify that it is good? That'd be the Github model; require approval when the code changes, but don't require peer approval to deploy / rollback the code after it's been merged. In that model we'd remove the approval when a recipe changes but not when it is disabled.

This probably needs some feedback from User Advocacy.